src/core/src/pass/visualize_tree.cpp

```diff
@@ -213,6 +215,16 @@ static void collect_symbol_print_values(const std::shared_ptr<ov::Model>& m,
 
 bool ov::pass::VisualizeTree::run_on_model(const std::shared_ptr<ov::Model>& f) {
     RUN_ON_MODEL_SCOPE(VisualizeTree);
+
+    static const bool ovasp = ov::util::getenv_bool("OV_VISUALIZE_APPLY_SYMBOLIC_PROPAGATION");
+    if (ovasp) {
+        std::cerr << "Warning: OV_VISUALIZE_APPLY_SYMBOLIC_PROPAGATION enabled. ov::pass::SymbolicPropagation will be "
```

Comment:
If it is a warning, could you be more specific about what we should be afraid of as a negative side effect?